docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
       - SITE_URL=http://backend:8000/
       - TREEHERDER_DEBUG=True
       - NEW_RELIC_INSIGHTS_API_KEY=${NEW_RELIC_INSIGHTS_API_KEY:-}
+      - HOSTNAME=${HOSTNAME:-}
```

Comment:
Does this work for you?

On Mac, I don't have HOSTNAME.
You can probably use the command `hostname` which is available on both Linus and Mac

Reply:
Okay!